ext/mbstring/tests/mb_language.phpt

```diff
@@ -0,0 +1,40 @@
+--TEST--
+mb_language()
+--SKIPIF--
+extension_loaded('mbstring') or die('skip');
```

Comment:
This needs a `<?php` (same in the other test). Also the function_exists check can be dropped. I don't think those functions are optional (if mbstring is enabled).

Reply:
This has been fixed.  Thanks for the feedback!